demo/src/checkout-pricing.js

```diff
@@ -151,10 +160,11 @@ function CheckoutPricingForm () {
       <div style={{ marginTop: '10px' }}>
         {recurlyError ? <span style={{ color: 'red' }}>{recurlyError.message}</span> : ''}
         {showPrice ? (
-          <span>
-            {'Subtotal: '}
-            <span>{`${price.currency.symbol}${price.now.subtotal}`}</span>
-          </span>
+          <>
+            <h2>{'Subtotal: '}</h2>
+            <div>Now: {`${price.currency.symbol}${price.now.subtotal}`}</div>
+            <div>Next: {`${price.currency.symbol}${price.next.subtotal}`}</div>
```

Comment:
These labels should be more expressive, as though a customer were the audience: "Subtotal due now" / "Subtotal due next billing cycle" -- and perhaps we should use total instead of subtotal?

Reply:
I'm wondering if we should revert this commit and open a new PR/issue to update the demo more broadly